templates/openapi-generator/model/ObjectSerializer.mustache

```diff
@@ -234,7 +234,10 @@ export class ObjectSerializer {
   /**
     * Parse data from a string according to the given media type
     */
-  public static parse(rawData: string, mediaType: string | undefined) {
+  public static parse(rawData: string, mediaType: string | undefined, dataType: string | null) {
+    if (dataType === 'void') {
```

Comment:
let's check with openapi-spec channel if the spec can be updated with `204` response

Reply:
https://github.com/okta/okta-oas3/pull/147